std/stdio.d

```diff
@@ -1368,6 +1368,32 @@ Throws: $(D Exception) if the file is not opened.
         w.put('\n');
     }
 
+    /**
+
+    Writes its arguments in text format to the file separated pairwise by a
+    space (or custom `separator`), followed by a newline (or custom `eol`).
+
+    Throws: $(D Exception) if the file is not opened.
+            $(D ErrnoException) on an error writing to the file.
+    */
+    void print(string separator = " ", string eol = "\n", S...)(auto ref S args)
+    {
+        import std.range : repeat;
+        import std.string : join;
+        static immutable fmt = repeat("%s", S.length).join(separator) ~ eol;
```

Comment:
This is over engineering. A static foreach loop for `args` and default `FormatSpec!char` along with `formatValue` would be faster to compile and faster to execute, and looks clear.